srcpkgs/vscode/template

```diff
@@ -35,18 +40,29 @@ do_build() {
 	_mem_limit="--max_old_space_size=4095"
 
 	export NODE_OPTIONS="${_mem_limit}"
-	yarn install --ignore-engines --arch=x64
+	/usr/lib/node_modules/npm/bin/node-gyp-bin/node-gyp install \
+		--target=$_electronver \
+		--tarball=/usr/include/electron7/node_headers.tar.gz
+	yarn install --ignore-engines --frozen-lockfile --arch=x64
+	export CFLAGS="$CFLAGS -I/usr/include/node"
 	yarn run gulp vscode-linux-x64-min
 }
 
 do_install() {
 	vmkdir usr/lib/code-oss
-	vcopy ../VSCode-linux-x64/* usr/lib/code-oss
+	vcopy ../VSCode-linux-x64/resources usr/lib/code-oss
+	vcopy ../VSCode-linux-x64/bin usr/lib/code-oss
 	vmkdir usr/bin
 	ln -sf /usr/lib/code-oss/bin/code-oss ${DESTDIR}/usr/bin/
 	vmkdir usr/share/applications
 	ln -sf /usr/lib/code-oss/resources/app/resources/linux/code.desktop ${DESTDIR}/usr/share/applications/code-oss.desktop
 	vmkdir usr/share/pixmaps
 	ln -sf /usr/lib/code-oss/resources/app/resources/linux/code.png ${DESTDIR}/usr/share/pixmaps/code-oss.png
+	ln -sf /usr/bin/rg ${DESTDIR}/usr/lib/code-oss/resources/app/node_modules.asar.unpacked/vscode-ripgrep/bin/rg
+	vsed \
+		-e "s|VSCODE_PATH=/usr/share/vscode|VSCODE_PATH=/usr/lib/vscode|g" \
```

Comment:
I think this should be /usr/lib/code-oss

Reply:
Well yes and no, we have a patch inplace that did already change VSCODE_PATH, so that line should just be removed. (I probably should just move everything into the patch).